genshi/template/astutil.py

```diff
@@ -729,15 +732,17 @@ def _process_slice(node):
                 if getattr(node, 'step', None):
                     self._write(':')
                     self.visit(node.step)
-            elif isinstance(node, _ast.Index):
+            elif isinstance(node, (_ast.Index, _ast_Constant)):
```

Comment:
I am guessing that `_ast_Constant` should have it's only elif that looks like:
```python
elif isinstance(node, _ast_Constant):
    self.visit_Constant(node)
```
However, I'm worried that now almost any kind of AST node could appear here and that instead we need something right at the end that does:
```python
elif isinstance(node, _ast.Node):
    self.visit(node)
```
Here `_ast.Node` is a place holder for a base class for all AST nodes. I'm not 100% such a base class exists.

Reply:
Good catch about `self.visit_Constant(node)`, will change that for sure.